### PR TITLE
[GLOSSARY] transaction surveilance company

### PR DESCRIPTION
**Pull request by MaxHillebrand**
adds transaction surveillance company to the glossary, the text is forked from the [separate chapter](/why-wasabi/TransactionSurveillanceCompanies.md).

### CODE REVIEW

File: docs/glossary/Glossary.md [hunk 1]
```diff
@@ -277,6 +277,14 @@ Tumbling is a synonym of 'Mixing'.
 Similarly, Tumbler is the synonym of 'Mixer'.
 :::
 
+:::details
+### Transaction Surveillance Company
+
+A transaction surveillance company is one which attempts to spy on all bitcoin users.
```
Comment (SimoneBovi) [inline docs/glossary/Glossary.md]: A transaction surveillance company is one which attempts to spy on all Bitcoin users.

Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]:
```suggestion
A transaction surveillance company is one which attempts to spy on all Bitcoin users.
```

**Comment by RiccardoMasutti** [comment]
Glossary should contain only medium-advanced bitcoin explanation. Not the basics.

**Comment by RiccardoMasutti** [comment]
NACK.
I think this is self-explanatory.

**Comment by raindogdance** [comment]
FWIW, I've never yet encountered the term "transaction surveillance company" anywhere else. It seems to me that "chain analysis company" or "blockchain analytics company" or something like that would sound more conventional in privacy context.

File: docs/glossary/Glossary.md [hunk 2]
```diff
@@ -277,6 +277,14 @@ Tumbling is a synonym of 'Mixing'.
 Similarly, Tumbler is the synonym of 'Mixer'.
 :::
 
+:::details
+### Transaction Surveillance Company
+
+A transaction surveillance company is one which attempts to spy on all Bitcoin users.
+Their business model is usually to sell the data to any governments, corporations and individuals willing to pay for their services.
```
Comment (yahiheb) [inline docs/glossary/Glossary.md]:
```suggestion
Their business model is usually to sell the data to any government, corporation or individual willing to pay for their services.
```

**Comment by MaxHillebrand** [comment]
> I think this is self-explanatory. - @RiccardoMasutti 

I think it is not really, it's a term not commonly used outside of Bitcoin...

**Pull request merged**